inc/post-templates.php

```diff
@@ -239,9 +283,9 @@ function largo_url_to_attachmentid($url) {
  * @filter largo_partial_by_post_type
  * @since 0.5.4
  */
-function largo_get_partial_by_post_type($partial, $post_type, $context) {
+function largo_get_partial_by_post_type( $partial, $post_type, $context ) {
 	// Remove this conditional in #926.
```

Comment:
I would rather do all the #926 stuff in its own PR.

Speaking of #926, why is it closed? As far as I know, we never actually went through and removed any of the argolinks code.